### PR TITLE
two issues

### LINKED ISSUE #2901: Contact Form Submission - Other
Someone submitted the contact form!

**URL**: https://usaco.guide/dashboard/
**Module**: Modular Arithmetic
**Topic**: Other
**Message**: 
for the editorial of Santa's bot, int would cause overflow and it will get WA on test 14. But it will get AC after you change it to long long.

### LINKED ISSUE #2909: Contact Form Submission - Unclear Explanation (VSCode)
Someone submitted the contact form!

**URL**: https://usaco.guide/general/running-code-locally?lang=cpp
**Module**: General - Running Code Locally
**Topic**: Unclear Explanation
**Message**: 
Visual Studio is an IDE, Visual Studio Code is a code editor.

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

closes #2909 and also closes #2901

### CODE REVIEW

File: solutions/cf-1279D.mdx [hunk 1]
```diff
@@ -77,10 +77,11 @@ int main() {
 	for (const vector<int>& c : children) {
 		for (const int& i : c) {
 			int num = 1 * 1 * wanter_nums[i];
```
Comment (envyaims) [inline solutions/cf-1279D.mdx:79]: ?

**Review by envyaims — APPROVED** [review]
ok ✌️

File: solutions/cf-1279D.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/cf-1279D.mdx:79]: think it's part of some math formula that happens to have 1's in it
makes things more explicit

**Review by ryanchou-dev — APPROVED** [review]
qwq

**Pull request merged**